File: coding-challenges/Week03/Day02_(04-02-2020)/Ques2.py
# Question- Write a Python script to generate and print a dictionary that contains a number (between 1 and n) in the form (x, x*x).

# Answer-

user_input = int(input("Enter a Number -> "))
d = dict()

for i in range(1 , user_input + 1):
	d[i] = i ** 2
print(d)

File: assignments/Week12/Day02_(07-04-2020)/Ques2.js
/*
Question 2) -  	Write a function fib() that takes an integer n and return the nth Fibonacci number.
*/

//Answer- 

function fib(n) {
  let a1 = [0, 1];

  if (n <= 2) return 1;

  for (let i = 2; i <= n; i++) {
    a1[i] = a1[i - 1] + a1[i - 2];
    // a1.push(a1[i-1] + a1[i-2])
  }

  return a1[n];
  // return a1
}

fib(7);